### PR TITLE
Updated README.md with Cassandra setup instructions.

### PR DESCRIPTION
**Pull request by jcjimenez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,40 @@ FACEBOOK_AUTH_TOKEN="..."
 
 FORTIS_FEATURE_SERVICE_HOST="..."
 ```
+
+## Development Setup
+In order to serve data from the GraphQL services, you will need to a functioning Cassandra installation.
```
Comment (c-w) [inline README.md:22]: Nit: extra `to`.

File: README.md [hunk 2]
```diff
@@ -17,3 +17,40 @@ FACEBOOK_AUTH_TOKEN="..."
...
+brew install cassandra
+brew services start cassandra
+```
+
+### First Casssandra Migration
+After installing a fresh copy of Cassandra, you will need to run the DDL initialization statements in:
+
+```
+curl https://raw.githubusercontent.com/CatalystCode/fortisdeploy/master/ops/storage-ddls/cassandra-setup.cql > cassandra-setup.cql
+cat cassandra-setup.cql | cqlsh
```
Comment (c-w) [inline README.md:54]: Nit: you can probably just curl straight into cqlsh.

**Review by c-w — APPROVED** [review]
LGTM.

File: README.md (hunk 2 above)
Comment (jcjimenez) [reply README.md:54]: Totally - I just like to let the user peek at to what they're about to execute.

**Pull request merged**